serenity-bdd/serenity-test-projects
# Added project that will be used in testing how junit and jbehave works when tests are run in parallel

#### Summary of this PR

Added project that will be used in testing how junit and jbehave works when tests are run in parallel
#### Intended effect

Base project with some structure will be used in tests in serenity-module-integrations to generate test classes and run them in parallel forks
#### How should this be manually tested?

`./gradlew clean test aggregate` should execute all build and generate all reports
#### Side effects

N/A
#### Documentation

N/A
#### Relevant tickets

serenity-bdd/serenity-jbehave/issues/45
#### Screenshots (if appropriate)

N/A
